src/foam/u2/wizard/agents/ReturnToLaunchPointAgent.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.agents',
+  name: 'ReturnToLaunchPointAgent',
+  flags: ['web'],
+  documentation: `
+  Handle route changes on wizard close
+  `,
+
+  imports: [
+    'currentMenu',
+    'ctrl'
+  ],
+
+  implements: [
+    'foam.core.ContextAgent'
+  ],
+
+  methods: [
+    async function execute() {
+      if ( this.currentMenu.id === this.ctrl.route ) {
+        window.history.back();
```

Comment:
Will this work if the wizard is the first thing in browser history?

Reply:
When is this not true?

Reply:
If you open a tab and go straight to the transferwizard, could be a link from an email or a direct link to onboarding